packages/server/graphql/queries/helpers/fetchAllRepoIntegrations.ts

```diff
@@ -11,13 +12,15 @@ const fetchAllRepoIntegrations = async (
   info: GraphQLResolveInfo
 ) => {
   const {dataLoader} = context
+
   const permLookup = await getPermsByTaskService(dataLoader, teamId, userId)
-  const [prevRepoIntegrations, jiraProjects, githubRepos] = await Promise.all([
+  const [prevRepoIntegrations, jiraProjects, githubRepos, jiraServerProjects] = await Promise.all([
     getPrevRepoIntegrations(userId, teamId, permLookup),
     dataLoader.get('allJiraProjects').load({teamId, userId}),
-    fetchGitHubRepos(teamId, userId, dataLoader, context, info)
+    fetchGitHubRepos(teamId, userId, dataLoader, context, info),
+    fetchJiraServerProjects(teamId, userId, dataLoader)
   ])
-  const fetchedRepoIntegrations = [...jiraProjects, ...githubRepos]
+  const fetchedRepoIntegrations = [...(jiraProjects ?? []), ...githubRepos, ...jiraServerProjects]
```

Comment:
+1 jiraProjects is guaranteed to be an array now so no need for the nullish coalescing